src/documents/readme.ts

```diff
@@ -39,73 +41,119 @@ export class Readme extends MarkdownDocument {
+      sortOrder: 0,
       builder,
-    );
-  }
-
-  public apiReference(builder: Builder<ApiReference>) {
-    return this.executeBuilder(this._apiReference, ApiReference, builder);
+    });
   }
 
-  public appendix(builder: Builder<Appendix>) {
-    return this.executeBuilder(this._appendix, Appendix, builder);
+  public environmentVariables(builder: Builder<EnvironmentVariables>) {
+    return this.executeBuilder({
+      type: 'environmentVariables',
+      sortOrder: 1,
+      builder,
+    });
   }
 
-  public authors(builder: Builder<Authors>) {
-    return this.executeBuilder(this._authors, Authors, builder);
+  public runLocally(builder: Builder<RunLocally>) {
+    return this.executeBuilder({
+      type: 'runLocal',
+      sortOrder: 2,
+      builder,
+    });
   }
 
-  public contributing(builder: Builder<Contributing>) {
-    return this.executeBuilder(this._contributing, Contributing, builder);
+  public roadmap(builder: Builder<Roadmap>) {
+    return this.executeBuilder({
+      type: 'roadmap',
+      sortOrder: 3,
+      builder,
+    });
   }
 
-  public environmentVariables(builder: Builder<EnvironmentVariables>) {
-    return this.executeBuilder(
-      this._environmentVariables,
-      EnvironmentVariables,
+  public apiReference(builder: Builder<ApiReference>) {
+    return this.executeBuilder({
+      type: 'apiReference',
+      sortOrder: 4,
       builder,
-    );
+    });
   }
 
   public examples(builder: Builder<Examples>) {
-    return this.executeBuilder(this._examples, Examples, builder);
+    return this.executeBuilder({
+      type: 'examples',
+      sortOrder: 5,
+      builder,
+    });
   }
 
   public faq(builder: Builder<FAQ>) {
-    return this.executeBuilder(this._faq, FAQ, builder);
+    return this.executeBuilder({
+      type: 'faq',
+      sortOrder: 6,
+      builder,
+    });
   }
 
-  public installation(builder: Builder<Installation>) {
-    return this.executeBuilder(this._installation, Installation, builder);
+  public support(builder: Builder<Support>) {
+    return this.executeBuilder({
+      type: 'support',
+      sortOrder: 7,
+      builder,
+    });
   }
 
-  public roadmap(builder: Builder<Roadmap>) {
-    return this.executeBuilder(this._roadmap, Roadmap, builder);
+  public appendix(builder: Builder<Appendix>) {
+    return this.executeBuilder({
+      type: 'appendix',
+      sortOrder: 8,
+      builder,
+    });
   }
 
-  public runLocally(builder: Builder<RunLocally>) {
-    return this.executeBuilder(this._runLocally, RunLocally, builder);
+  public authors(builder: Builder<Authors>) {
+    return this.executeBuilder({
+      type: 'authors',
+      sortOrder: 9,
+      builder,
+    });
   }
 
-  public support(builder: Builder<Support>) {
-    return this.executeBuilder(this._support, Support, builder);
+  public acknowledgements(builder: Builder<Acknowledgements>) {
+    return this.executeBuilder({
+      type: 'acknowledgements',
+      sortOrder: 10,
```

Comment:
part of wonders whether the sort order default could be brought out to an Enum/Object so that they can be seen in one place. I am not sure if it is necessary so see what you think.

Reply:
This is a good point, we could take them out into a set based on type and then base the sort order on index so that reordering is easier.